### ISSUE TITLE
Add "Skip to" link styling and documentation for accessibility

### ISSUE THREAD

**Issue by jeremyBass** [washingtonstateuniversity/WSU-spine#18]
We should put in some things for what WCAG and Sec. 508 needs.  There are many things from http://accessibility.psu.edu/skipnav, to telling both the blind and search engines where the main nav is and content is.  For example:

```
<ul class="accessibility">
    <li><a href="#spine-navigation" title="Skip to navigation" accesskey="n">Skip to navigation</a></li>
    <li><a href="#main" title="Skip to content" accesskey="s">Skip to content</a></li>
</ul>
<hr class="accessibility" />
```

This allows good accessibility to the user that needs it.  There is much more, but I think we should be including this in our best practice examples and anything that we are generating.

**Comment by jeremyfelt** [comment]
I think we can approach this in a few parts:

1. Provide styling for a class that puts any skip to links off screen until focused.
2. Provide the ideal markup in our `spine.html` and `spine.min.html` files.
3. Provide documentation at web.wsu.edu for best practices.

**Comment by jeremyfelt** [comment]
This is now part of #418.
